### DISCUSSION TITLE
[SUGGESTION] AI P.E.T Combat/AI fleet command

### DISCUSSION THREAD

**Discussion by Reason147** [tsunamayo/Starship-EVO#4009] [Suggestions]
Something I think would be a great idea to add at some point is AI P.E.T Combat or AL fleet command, It would be nice to have fleets of fighters, drones or ships in a fleet commanded by a command ship for combat or AI fleets, Just an Idea.

**Comment by TheShadowSage** [comment]
> Something I think would be a great idea to add at some point is AI P.E.T Combat or AL fleet command, It would be nice to have fleets of fighters, drones or ships in a fleet commanded by a command ship for combat or AI fleets, Just an Idea.

Are you thinking of creating some kinda "tactical yoke"? Cause I'm imaging you come up to a computer yoke, and it has a whole bunch of buttons  (i.e. More Hotkeys!) that you can press for various things like say:

- Register a ship (Allows player to register any ship to their fleet)

- Fleet Command (This button will show your entire fleet along with other actions for moving, attacking, scavenging, etc.)

- Hire Crew (This window will show you what sort of personal you need to operate, repair, and upgrade your ships )

- Exit (Exit Computer Yoke)

So when you get the Fleet Command window it would give you an overlay like in Star Wars : Empire at War GUI that shows you your ships on the holomap with a list of actions you can tell them from attacking NPCs, AI, or getting you resources.
